### PR TITLE
[ZEPPELIN-2447] Fix python interpreter as to use max result setting

### PR DESCRIPTION
**Pull request by paularmand**
### What is this PR for?
zeppelin_python.py used a hard-coded 1000 rows limit when returning results. This PR makes sure that the interpreter retrieves the max_results from PythonInterpreter.java and that the latter retrieves this configuration from the configuration via the properties object.

### What type of PR is it?
Bug Fix

### Todos
* [ ] - Task

### What is the Jira issue?
https://issues.apache.org/jira/browse/ZEPPELIN-2447

### How should this be tested?
+ use %python interpreter
+ make a dataframe with more than a 1000 rows
+ z.show the dataframe
+ export it in csv 
+ the numbers of rows should exceed 1000 rows

### Questions:
* Does the licenses files need update?
no
* Is there breaking changes for older versions?
no
* Does this needs documentation?
no

### CODE REVIEW

**Comment by paularmand** [comment]
Ready for review

File: python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java [hunk 1]
```diff
@@ -96,6 +96,7 @@
 
   public PythonInterpreter(Properties property) {
     super(property);
+    this.maxResult = Integer.parseInt(property.getProperty(MAX_RESULT));
```
Comment (zjffdu) [inline python/src/main/java/org/apache/zeppelin/python/PythonInterpreter.java]: Use `property.getProperty(MAX_RESULT, default_value)`

**Comment by zjffdu** [comment]
ping @paularmand Can you fix the comment ?

**Comment by paularmand** [comment]
@zjffdu Thanks for review. Added 1000 hard coded as default for now, although I believe a better solution would be to add fallbacks to a property class upfront (see https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#defaults).

**Comment by zjffdu** [comment]
Thanks @paularmand LGTM

**Comment by zjffdu** [comment]
@paularmand Could you add jira to the title of your PR ? name it as `[ZEPPELIN-2447] Fix python interpreter as to use max result setting`

**Comment by paularmand** [comment]
Is the Jenkins failure an unrelated selenium test failure, or do I have to look into it?

**Comment by paularmand** [comment]
ping @zjffdu Do I have to look into it the Selenium test failure?

**Comment by zjffdu** [comment]
@paularmand Selenium test is flaky, one thing left is unit test. Could you add unit test in `PythonInterpreterTest` ？

**Comment by paularmand** [comment]
@zjffdu I'll have a look. Can expect something later on this week.

**Comment by jongyoul** [comment]
ping.

**Comment by paularmand** [comment]
Pong @jongyoul . I wrote the unit test, but it is not passing on Jenkins. I have trouble setting up my local IDE to run the python tests in IntelliJ (all other tests, except the python ones, are running fine), so I'll have to dig deeper in the IDE settings. Pointers/help appreciated.

**Comment by jongyoul** [comment]
Let me check

**Pull request closed without merging**